### ISSUE TITLE
Custom response structure not worked

### ISSUE THREAD

**Issue by krek95** [MarcinOrlowski/laravel-api-response-builder#155]
**What's wrong?**
After installing the package, I tried to change the response. But after call custom class method I get a standard "ResponseBuilder" response.

My custom class:
```
namespace App\Builders;

use Carbon\Carbon;
use MarcinOrlowski\ResponseBuilder\ResponseBuilder;

class ApiResponseBuilder extends ResponseBuilder
{
    protected function buildResponse(bool $success, int $api_code,
                                            $message_or_api_code, array $lang_args = null,
                                            $data = null, array $debug_data = null) : array
    {
        $response = parent::buildResponse($success, $api_code, $message_or_api_code, $lang_args, $data, $debug_data);

        $response['server_time'] = Carbon::now();

        unset($response['locale']);

        return $response;
    }

}
```

ApiResponseBuilder::success() returns following:
```
{
    "success": true,
    "code": 0,
    "locale": "en",
    "message": "OK",
    "data": null
}
```

**Steps to reproduce the behavior:**
1. install package
2. create custom response class

**Runtime environment**
 - Name and version of OS: Windows 10
 - PHP version 7.3
 - Laravel version 8.10.0

**Comment by MarcinOrlowski** [comment]
Your code is correct. This is a bug in `ResponseBuilder` and I have fix in tests now. Thanks for the report.

**Comment by MarcinOrlowski** [comment]
This is now fixed in v8.1.1.
